easybuild/easyconfigs/i/icc/README.md

```diff
@@ -0,0 +1,13 @@
+In order to use these configs you must first download the Intel Parallel Studio XE CPP file. NOT the full bundle! The file should look like this: parallel_studio_xe_2018_update2_composer_edition_for_cpp.tgz    
+
+Next you must specify the license file. There are three options:  
+* Place it in the following path: ~/licenses/intel/license.lic
+* Set an environment variable: `export INTEL_LICENSE_FILE=/path/to/file.lic`
+* Set an environment variable: `export LM_LICENSE_FILE=/path/to/file.lic`
+
+To install, you must past eb the sourcepath to find the download. There are a number of ways to do this assuming the file is in ~/EB_Downloads.  
+* Pass it as a option on the command line: `eb icc-2018.2.199-GCC-6.4.0-2.28.eb -r --sourcepath=/home/username/EB_Downloads`  
```

Comment:
you can use `~/EB_Downloads` here?

Reply:
No. I thought I could short cut it, but it didn't work when I tested and I updated the other places, but missed this one.

Reply:
``--sourcepath ~/EB_Downloads` would work, since then `bash` will expand the `~`, it maybe won't when you're using ``--sourcepath=...`